controllers/dspipeline_params.go

```diff
@@ -87,6 +86,10 @@ func (p *DSPAParams) UsingV2Pipelines(dsp *dspa.DataSciencePipelinesApplication)
 	return dsp.Spec.DSPVersion == "v2"
 }
 
+func (p *DSPAParams) UsingV1Pipelines(dsp *dspa.DataSciencePipelinesApplication) bool {
+	return dsp.Spec.DSPVersion == "v1" || dsp.Spec.DSPVersion == ""
```

Comment:
A little more bullet-proof/concise way to do this:

```suggestion
	return !p.UsingV2Pipelines(dsp)
```

(basically, if its *not* v2 then it has to be v1, whereas the current changeset allows a gray area where something could be neither v1 or v2, and as such UsingV1Pipelines=false, but UsingV2Pipelines=false as well)